config/e2e-dev.yml

```diff
@@ -0,0 +1,25 @@
+analysis:
+  analyzers:
+    - name: consensus_main_damask
+      chain_id: oasis-3
```

Comment:
Does this have to be `oasis-3`? Could it be something like `oasis-e2e-test`? Really, anything that's not a chain ID we plan to index for real, so that it'll catch any errors where we're hardcoding (and not being generic enough with passing chain ID as a parameter).

Reply:
`oasis-test` looks okay